Tools/ci/metadata_failsafe_web.sh

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env bash
+#
+# metadata_failsafe_web.sh — build and sync failsafe webapp metadata files
+#
+# Usage:
+#   Tools/ci/metadata_failsafe_web.sh [--test-only] [--debug]
+#
+# Options:
+#   --test-only   Run build and comparison; exit 1 if diffs found, without copying files
+#   --debug       Show full build output and debug info for file comparisons and EMSDK install
+#
+set -euo pipefail
+shopt -s nullglob
+
+# Parse flags
+test_only=false
+debug=false
+while [[ $# -gt 0 ]]; do
+  case "$1" in
+    --test-only) test_only=true; shift ;;
+    --debug)     debug=true; shift ;;
+    *) echo "Usage: $0 [--test-only] [--debug]"; exit 2 ;;
+  esac
+done
+
+# Paths and commands
+build_cmd="make failsafe_web"
+src_dir="build/px4_sitl_default_failsafe_web"
+dest_dir="docs/public/config/failsafe"
+
+# Ensure Emscripten SDK is available
+if ! command -v emcc >/dev/null 2>&1; then
+  echo "🔧 Emscripten not found. Ensuring EMSDK is installed."
```

Comment:
I ran the this with `--test-only`, which installed the SDK, and then without. Still got this message (even though SDK is now installed)